sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -822,15 +824,19 @@ def _from_normal_time(self, value):
 
   def encode_to_stream(self, value, out, nested):
     # type: (IntervalWindow, create_OutputStream, bool) -> None
-    typed_value = value
+    if not TYPE_CHECKING:
```

Comment:
We should not be checking this at every element encoding; instead lift the import.